src/styles/_breakpoints.scss

```diff
@@ -0,0 +1,20 @@
+@mixin sm {
+    @media (max-width: 550px) {
+      @content;
+    }
+  }
```

Comment:
you defined `rwd` mixin at the bottom - maybe it is worth to use it inside `sm`, `md` and `lg` mixins?